projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -81,15 +81,14 @@ export class PageTimeRangePreferenceService {
   public getDefaultTimeRangeForCurrentRoute = (): TimeRange => {
     const currentRoute: ActivatedRoute = this.navigationService.getCurrentActivatedRoute();
     // Right side for when FF is enabled but 'defaultTimeRange' is not set on AR data
+
     return currentRoute.snapshot.data?.defaultTimeRange ?? this.getGlobalDefaultTimeRange();
   };
 
-  public getGlobalDefaultTimeRange = (): TimeRange => {
-    return new RelativeTimeRange(new TimeDuration(1, TimeUnit.Hour));
-  };
+  public getGlobalDefaultTimeRange = (): TimeRange => new RelativeTimeRange(new TimeDuration(1, TimeUnit.Hour));
```

Comment:
nit: now that using fat arrows, this and `getDefaultTimeRangeForCurrentRoute` can be switched back to standard prototype methods.